### DISCUSSION TITLE
봇이 지금 초기화중이라고만 뜨고 아예 안읽어줍니다

### DISCUSSION THREAD

**Discussion by raytv3951** [Lastorder-DC/chatreader-kor#75] [문의]
![스크린샷 2023-09-29 203139](https://github.com/Lastorder-DC/chatreader-kor/assets/146440745/e7a3354f-6dc4-4df7-aa5d-0ec302206a33)
이렇게 뜨고 일을 안해요

**Comment by Lastorder-DC** [comment]
트위치 로그인이 간혹 풀린 경우 발생할수 있습니다. 로그아웃후 로그인해보세요.
